### DISCUSSION TITLE
yq now breaks formatting of file?

### DISCUSSION THREAD

**Discussion by pspurlock07** [mikefarah/yq#1210] [Q&A]
Not sure when this started, but yq now does not preserve formatting in a file. As an example:

```
some: stuff

more:
  - things
     are
     here
```

Will turn into this:

```
some: stuff
more:
  - things are here
```

Is there a flag I'm missing, or has this changed?

**Comment by andry81** [comment]
Different implementations of `yq` rely on dependencies. The problem is much bigger than just blank lines. Here is a workaround through the diff+patch method which detects any change including an unexpected:

https://stackoverflow.com/questions/57627243/how-to-prevent-yq-removing-comments-and-empty-lines/72041655#72041655
